frontends/web/src/routes/exchange/pocket.tsx

```diff
@@ -175,6 +176,67 @@ export const Pocket = ({ code }: TProps) => {
     });
   };
 
+  const handlePaymentRequest = useCallback(async (message: PaymentRequestV0Message) => {
+    if (!message.slip24) {
+      alertUser(t('unknownError', { errorMessage: 'Missing payment request data' }));
+      return;
+    }
+
+    // this allows to correctly handle sats mode
+    const parsedAmount = await parseExternalBtcAmount(message.amount.toString());
+    if (!parsedAmount.success) {
+      alertUser(t('unknownError', { errorMessage: 'Invalid amount' }));
+      return;
+    }
+
+    const txInput: TTxInput = {
+      address: message.bitcoinAddress,
+      amount: parsedAmount.amount,
+      feeTarget: 'high',
+      customFee: '',
+      sendAll: 'no',
+      selectedUTXOs: [],
+      paymentRequest: message.slip24,
+    };
+
+    let result = await proposeTx(code, txInput);
```

Comment:
Not sure but you probably have to wipe the proposed tx note afterwards (maybe in a finally{} clause?), to avoid the bug that was fixed in https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2739

Reply:
Could not reproduce the bug, but done anyway. It can't harm